io-engine/src/bdev/nexus/nexus_bdev_rebuild.rs

```diff
@@ -462,7 +465,10 @@ impl<'n> Nexus<'n> {
             }
         }
 
-        self.reconfigure(DrEvent::ChildRebuild).await;
+        self.reconfigure(DrEvent::ChildRebuildEnd(
+            self.get_child_device_name(child_uri)?,
```

Comment:
hmm what happens if we fail here, we don't reconfigure channels is this ok?

Reply:
Yes it should be ok because that would mean device is unplugged underneath us and reconfiguration would've happened. If the device is not found for any other reason, I think we would already have a broken system in other ways.

Reply:
Just wanted to clarify because now with this change, there's nothing to fixup the channels, if we somehow "leak" a handle in the lists, there's no more reconconfigure_all that would remove them.